payjoin/src/core/receive/v1/mod.rs

```diff
@@ -34,17 +34,16 @@
 use std::cmp::{max, min};
 
 use bitcoin::psbt::Psbt;
-use bitcoin::secp256k1::rand::seq::SliceRandom;
 use bitcoin::secp256k1::rand::{self, Rng};
 use bitcoin::{Amount, FeeRate, OutPoint, Script, TxIn, TxOut, Weight};
 use serde::{Deserialize, Serialize};
 
 use super::error::{
-    InputContributionError, InternalInputContributionError, InternalOutputSubstitutionError,
-    InternalSelectionError,
+    InputContributionError, InternalInputContributionError, InternalSelectionError,
 };
 use super::optional_parameters::Params;
 use super::{InputPair, OutputSubstitutionError, ReplyableError, SelectionError};
+#[cfg(feature = "v1")]
 use crate::output_substitution::OutputSubstitution;
```

Comment:
If you want to export this as `receive::v1::OutputSubstitution` the right place to do that is in /exclusive/mod.rs because everything exported there is re-exported as

```rs
#[cfg(feature = "v1")]
pub use exclusive::*;
```